plugins/football-scoreboard/scroll_display.py

```
@@ -1,679 +1,310 @@
-        """Clear all scroll displays."""
-        for scroll_display in self._scroll_displays.values():
-            scroll_display.clear()
-        self._current_game_type = None
 
 
+    class ScrollDisplayManager(_ScrollDisplayManagerBase):
+        """Football scroll manager -- everything but the extras below is core."""
+
+        display_class = ScrollDisplay
+
+        def get_dynamic_duration(self, game_type: str = None) -> int:
```

Ruff (0.16.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)